scripts/comparison/compare_results.py

```diff
@@ -4,19 +4,22 @@
 import getopt
 import os
 import xml.etree.ElementTree as ET
-from html_report import htmlReportSection, htmlReport
+from html_report import HtmlReportSection, HtmlReport
 import glob
+import configparser
 
 
-def getParameters(argv):
+def get_parameters(argv):
     outputFile = ''
+    iniFile = ''
     try:
-        opts, args = getopt.getopt(argv, "p:u:o:",
```

Comment:
there is a better way for python to parse args: https://docs.python.org/2/howto/argparse.html